app/javascript/components/subnet-form/index.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import MiqFormRenderer from '@@ddf';
+import createSchema from './subnet-form.schema';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+import { API } from '../../http_api';
+
+const SubnetForm = ({ recordId }) => {
+  const [{ initialValues, isLoading, fields }, setState] = useState({ isLoading: !!recordId, fields: [] });
+  const submitLabel = !!recordId ? __('Save') : __('Add');
+
+  const loadSchema = (appendState = {}) => ({ data: { form_schema: { fields } } }) => {
+    if (!!recordId && appendState.initialValues.type === 'ManageIQ::Providers::Openstack::NetworkManager::CloudSubnet') {
+      Object.assign(fields[0], {isDisabled: true});
+      Object.assign(fields[1], {isDisabled: true});
+      Object.assign(fields[4], {isDisabled: true});
```

Comment:
This is needed since these fields are coming from the provider. There's also a chance that other providers may need similar treatment but I can't test those at the moment.